fairworkflows/nanopub.py

```diff
@@ -23,7 +23,7 @@ class Nanopub:
     PROV = rdflib.Namespace("http://www.w3.org/ns/prov#")
     DUL = rdflib.Namespace("http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#")
     BPMN = rdflib.Namespace("http://dkm.fbk.eu/index.php/BPMN2_Ontology#")
-    PWO = rdflib.Namespace("http://purl.org/spar/pwo#")
+    PWO = rdflib.Namespace("http://purl.org/spar/pwo/")
```

Comment:
See https://sparontologies.github.io/pwo/current/pwo.html, this seems like how the URIs are constructed right (instead of #)?

Reply:
Ah yes, that was a mistake for pwo. But note that many of the ontology URIs do indeed use ```#```. There isn't a consistent form, unfortunately.

Reply:
Check